api/geth_backend.go

```diff
@@ -2294,6 +2299,19 @@ func (b *GethStatusBackend) GetActiveAccount() (*multiaccounts.Account, error) {
 	return b.account, nil
 }
 
+func (b *GethStatusBackend) LocalPairingStarted() error {
+	if b.account == nil {
+		return errors.New("master key account is nil in the GethStatusBackend")
+	}
+
+	accountDB, err := accounts.NewDB(b.appDB)
+	if err != nil {
+		return err
+	}
+
+	return accountDB.MnemonicWasShown()
```

Comment:
Covered in [this](https://github.com/status-im/status-go/pull/4325#issuecomment-1831926033).